README.md

```diff
@@ -1 +1,52 @@
-# Hello-Microverse_
\ No newline at end of file
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
+> Description the project.
+This project is configuration of linter into html and CSS.
+
+## Built With
+
+- Major languages
+HTML and CSS
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+You need to have project prerequistes :
+
+HTML & CSS knowledge.
+Use linter for HTML & CSS.
+
+
+
+## Authors
+
+👤 **Abdifatah**
+
+- GitHub: [@AbdifatahYasin1](https://github.com/AbdifatahYasin1)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/cabdifataax-yaasiin-69977019a/)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+Microverse.
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
+
+=======
+# Hello Microverse
```

Comment:
- [ ] As I said before: Please 🙏🏽 delete this part from the README file to look similar to the [template](https://github.com/microverseinc/readme-template). The README file ends with the LICENSE section.